guide/popular-topics/permissions-extended.md

```diff
@@ -79,6 +79,7 @@ During your development, you will likely run into `DiscordAPIError: Missing Perm
 - It is trying to execute a forbidden action on the server owner.
 - It is trying to execute an action based on another unfulfilled factor (for example, reserved for partnered guilds).
 - It is trying to execute an action on a voice channel without the `ViewChannel` permission.
+- It is trying to execute an action on a member/channel/role whilst timed out
```

Comment:
Appreciate the consistency, but I'd prefer some other phrasing that's easier to comprehend for people that struggle with english a bit (avoiding whilst, mostly)

Reply:
I'll keep the first half in line with the rest. Second-half could be made a bit simpler with something like
> It is trying to execute an action on a member, channel, or role while the bot is timed out

Reply:
does it just apply to those? thought it was just about anything that's prevented

Reply:
I'll quickly check that actually. I know from personal experience all those are effected, but it likely is more. Idk about wording for everything. Ig it could be trying to execute an action *within a guild* while the bot is timed out? Not sure

Reply:
or straight-up
```suggestion
- Your bot is currently timed out.
```

Reply:
Yeah fair enough. Suppose that's the simplest approach